lisa/mixin_modules.py

```diff
@@ -62,6 +62,12 @@
     except ModuleNotFoundError as e:
         print(f"libvirt package is not installed. [{e}]")
 
+try:
+    import lisa.sut_orchestrator.hyperv.platform_  # noqa: F401
+    import lisa.sut_orchestrator.hyperv.schema  # noqa: F401
```

Comment:
I think the schema file is referenced in platform, so it doesn't need to be listed here.

Reply:
Fixed.